src/main.rs

```diff
@@ -32,9 +32,17 @@ mod weekly;
 use actix::Actor;
 use failure::ResultExt;
 use log::{info, trace};
+use prometheus::IntGauge;
 use structopt::clap::{crate_name, crate_version};
 use structopt::StructOpt;
 
+lazy_static::lazy_static! {
+    static ref PROCESS_START_TIME: IntGauge = register_int_gauge!(opts!(
+        "process_start_time_seconds",
+        "UTC start time of the process in seconds."
```

Comment:
Minor nit: I'd clarify that this is an offset from the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time).

The chrono docs say:

> Returns the number of non-leap seconds since January 1, 1970 0:00:00 UTC (aka "UNIX timestamp").

How about something like:

`Time the process started as non-leap seconds since the "Unix epoch" in UTC`

Reply:
The recommend description for this gauge is `Start time of the process since unix epoch in seconds.`, as per https://prometheus.io/docs/instrumenting/writing_clientlibs/#process-metrics.

Reply:
Thanks for review. I will change to clarify Unix epoch per Prometheus docs.